ports/servoshell/desktop/headed_window.rs

```diff
@@ -443,7 +443,7 @@ impl Window {
         handled
     }
 
-    #[allow(unused_variables)]
+    #[expect(unused_variables)]
```

Comment:
The variable is used on macos, so this would need to be conditional on the target_os